### DISCUSSION TITLE
Enabled for [Site Name] Expanded by Default in Multi-Site

### DISCUSSION THREAD

**Discussion by ademers** [craftcms/cms#11304] [Ideas] [answered]
I think that it would be a better AX if **Enabled for [Site Name]** was expanded by default. It's easy for content authors to miss. If someone has an install with dozens of sites, they can collapse the list.

Before:
![CleanShot 2022-05-24 at 11 06 04@2x](https://user-images.githubusercontent.com/901095/170069102-6e306d59-3965-4504-9c35-3786b1410f69.png)

After:
![CleanShot 2022-05-24 at 11 07 44@2x](https://user-images.githubusercontent.com/901095/170069455-fd4c384e-982e-4541-93c7-34e64d7a1c27.png)

Thanks for your consideration.

**Comment by brandonkelly** [comment, marked as answer]
I’ve avoided this because there are a lot of installs with several (50+ even) sites, so expanding it by default would completely take over the page. Maybe we could just expand it if there are less than five sites though.

**Reply by ademers** [reply]
Yes. I like the idea of expanded if less than 5 or whatever number of sites. Thanks!

**Comment by TheFunkyMonk** [comment]
I came here to request this feature as well. Especially with sections set to allow entries to set their own sites and sites having a mix of entries, it's not very clear to the author what type of entry they're dealing with.

Perhaps this could be a general config option we could set if we know we're only dealing with a few sites and this is desired?
